<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" colspan="3">
     <widget class="QSlider" name="horizontalSlider">
      <property name="value">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>realtime / 3:41</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" colspan="5">
     <widget class="QSplitter" name="logSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="infoSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="rplNodeInfoTree">
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>data</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Rpl Instance</string>
         </property>
         <item>
          <property name="text">
           <string>Instance</string>
          </property>
          <property name="text">
           <string>50</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>DODAG</string>
         </property>
         <item>
          <property name="text">
           <string>Version</string>
          </property>
          <property name="text">
           <string>240</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="text">
           <string>aaaa::1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Prefix</string>
          </property>
          <property name="text">
           <string>aaaa::/16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MinHopRankInc</string>
          </property>
          <property name="text">
           <string>256</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Node</string>
         </property>
         <item>
          <property name="text">
           <string>WPAN address</string>
          </property>
          <property name="text">
           <string>1274002101929</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Link-local IPv6</string>
          </property>
          <property name="text">
           <string>fe80::02101929</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Global IPv6</string>
          </property>
          <property name="text">
           <string>aaaa::02101929</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rank</string>
          </property>
          <property name="text">
           <string>1291</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ETX</string>
          </property>
          <property name="text">
           <string>3.41</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sent packet count</string>
          </property>
          <property name="text">
           <string>141</string>
          </property>
         </item>
        </item>
       </widget>
       <widget class="QGraphicsView" name="rplTreeView"/>
      </widget>
      <widget class="QTextEdit" name="logEdit">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/icons/resources/icons/media-playback-start.png</normaloff>:/icons/resources/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start sniffing traffic</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/icons/resources/icons/media-playback-stop.png</normaloff>:/icons/resources/icons/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop sniffing traffic</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
